### PR TITLE
ADK final(?) pass on the manuscript

### PR DESCRIPTION
**Pull request by andrewkern**
okay here's my pass @petrelharp

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -220,9 +220,9 @@ \section*{Introduction}
 realistic simulations for population genetic analysis a simple matter of
 choosing pre-implemented models from a community-maintained catalog.
 The \stdpopsim catalog currently contains six species: humans,
-\textit{Pongo abelii},
+\textit{Pongo abelii}, \textit{Canis familiaris},
```
Comment (andrewkern) [inline popSimManu.tex:223]: this ordering of organisms in the manuscript was driving me nuts. see https://github.com/popsim-consortium/stdpopsim/issues/531

File: popSimManu.tex [hunk 2]
```diff
@@ -430,6 +431,7 @@ \subsection*{Simulation engines}
 especially in the presence of selection.
 However, since many sequence-based measures of population diversity remain nearly unchanged when rescaling in this fashion,
 this practice is effective for many purposes and widely employed.
+%ADK: I think we need a citaiton above for scaling. Maybe https://doi.org/10.1534/genetics.113.156935
```
Comment (andrewkern) [inline popSimManu.tex:431]: we need a citation here I think

File: popSimManu.tex [hunk 3]
```diff
@@ -791,23 +792,26 @@ \section*{Discussion}
...
-%and I don't think this is justified.
-Our current set may also omit other widely used models that can be readily incorporated into \stdpopsim.
-Additional complications such as changing
-genome builds as assemblies improve must also be incorporated.
-As the collection of models and species grows, we hope to
-incorporate crowd-sourced community input
-to highlight the most appropriate models and genome annotations.
+will be easily incorporated, as they are published.
+
+%ADK: I cut this off here-- I think thais is a fine ending
```
Comment (andrewkern) [inline popSimManu.tex]: Made a sort of larger cut at the end of the discussion here... hoping to get some 👀 on this

**Comment by andrewkern** [comment]
just did find/replace for "data is" -> "data are" and found a single instance in response to issue #199

File: popSimManu.tex (hunk 2 above)
Comment (petrelharp) [reply popSimManu.tex:431]: We cite that paper two sentences back, so I think it's fine.

File: popSimManu.tex [hunk 4]
```diff
@@ -393,6 +393,7 @@ \subsection*{The species catalog}
 
 
 %DRS: do we want to include these generic models in Table 1?
+%ADK: not a bad idea...
```
Comment (petrelharp) [inline popSimManu.tex]: We discussed this; most of the columns don't apply.

File: popSimManu.tex (hunk 3 above)
Comment (petrelharp) [reply popSimManu.tex]: This bit was explicitly in here as a response to a reviewer request; I may put some of it back in.

Comment (petrelharp) [reply popSimManu.tex]: Adding
"
Additional aspects of the framework,
such as genome builds, will also continue to change
as improvements are made to our understanding of genome structure.
"

**Review by petrelharp — APPROVED** [review]
LGTM! I've made one minor change.

**Comment by andrewkern** [comment]
i like these edits @petrelharp. i'm inclined to merge this at this point and we can get other eyes on the response doc.

**Pull request merged**